scripts/local_setup.sh

```diff
@@ -71,11 +81,12 @@ PASSWORD=$5
 echo "Using environment file: $ENV_FILE"
 
 # Main execution starts here
+check_druid
 setup_virtualenv
 load_env_vars
 run_pipeline
```

Comment:
The indexing steps needs the postgres database to be running, but if this is the first time you run the script, the postgres database hasn't yet started up.
You'll have to put init_db ahead of run_pipeline.

Reply:
Sure, I will do this